spec/helpers/questions_helper_spec.rb

```diff
@@ -1,6 +1,7 @@
 RSpec.describe QuestionsHelper, type: :helper do
   before do
-    allow(helper).to receive(:questions_to_ask).and_return(%w(question_1 question_2))
+    allow(helper).to receive(:session_questions).and_return(%w(question_1 question_2))
+    allow(helper).to receive(:all_questions).and_return(%w(question_1 question_2))
```

Comment:
By stubbing this, the [`all_questions`](https://github.com/alphagov/govuk-coronavirus-find-support/pull/125/files#diff-0890587450fdcfbd522211e4d850c710R57) method isn't tested. Perhaps we could use real questions?

Alternatively we could stub `I18n.t("coronavirus_form.groups")`. Perhaps a before step like this (I'm not sure it'll work):

```ruby
   en = { coronavirus_form: { groups: [] } }
   I18n.backend.store_translations(:en, en)
```

Reply:
Good point 👍.  I've updated to use real questions, which means `all_quesions` is no longer stubbed.